.github/workflows/build_python.yml

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Python (master, Hadoop 3, JDK 17, Scala 2.13)"
```

Comment:
Should probably add a Python version here too so we can easily distinguish it GitHub Actions tab.

Reply:
Sure. Let me use `Python 3.8/PyPy (master)" because the existing title was not visible already. 
![Screenshot 2023-10-03 at 11 38 56 PM](https://github.com/apache/spark/assets/9700541/679a4dc7-aca5-4f1b-ab93-43d978a78aa0)

We can revise this later when we add Python 3.10/3.11/3.12 later.